docs/qcfractal/source/setup_compute.rst

```diff
@@ -27,9 +27,14 @@ libraries while Dask requires ``dask``, ``dask.distributed``, and
 Using the Command Line
 ----------------------
 
+.. note::
+
+    The CLI + YAML config file is the current recommended way to start and run
+    Fractal Queue Managers
```

Comment:
We sometimes call this by different names. Can we standardize to `FractalManager` or similar? I think somewhere we had tried to do this.